examples/2d/elixir_euler_unstructured_quad_periodic.jl

```diff
@@ -9,21 +10,25 @@ equations = CompressibleEulerEquations2D(1.4)
 
 initial_condition = initial_condition_convergence_test
 source_terms = source_terms_convergence_test
-boundary_conditions = boundary_condition_periodic
+boundary_conditions = Dict( "Bottom" => boundary_condition_periodic,
+                            "Top"    => boundary_condition_periodic,
+                            "Right"  => boundary_condition_periodic,
+                            "Left"   => boundary_condition_periodic )
```

Comment:
Can't you keep the version you had before here?

Reply:
Sure, I was playing with trying to get the dictionary to recognize the `BoundaryConditionPeriodic` in the boundary flux routine. I can revert it